src/app/screen-snippet-handler.ts

```diff
@@ -235,30 +235,9 @@ class ScreenSnippet {
       if (this.focusedWindow && windowExists(this.focusedWindow)) {
         this.focusedWindow.moveTop();
       }
-      // remove tmp file (async)
-      if (this.outputFileName) {
-        this.deleteFile(this.outputFileName);
-      }
```

Comment:
Why do we need to remove this logic? If we don't delete the temp file, users' systems will start filling up. And, especially on VDIs, it can become a problem over time.